tsconfig.json

```diff
@@ -11,9 +11,9 @@
     "target": "es5",
     "declaration": true,
     "sourceMap": true,
-    "noImplicitAny": true,
```

Comment:
Any particular reasons to remove that? I guess not having implicit `any` types is supposed to be good?

Reply:
The reason for removing that was only to match what we currently have on `kogito-tooling` repo but I can put it back.

Reply:
Oh..... I didn't know we removed that from `kogito-tooling`. We should probably put it back :D